sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -49,15 +49,17 @@ def default_job_server(self, options):
     flink_master = self.add_http_scheme(
         flink_options.flink_master)
     flink_options.flink_master = flink_master
-    if flink_master in MAGIC_HOST_NAMES or sys.version_info < (3, 6):
-      return job_server.StopOnExitJobServer(FlinkJarJobServer(options))
-    else:
+    if (flink_options.flink_submit_uber_jar
+        and sys.version_info >= (3, 6)
```

Comment:
Should we fail when `flink_submit_uber_jar` is set and we use Python < 3.6? Otherwise we silently ignore the new flag.

Reply:
I suppose we should also consider both, python version and `flink_master`.

Reply:
Added comments and error for Python < 3.6.